src/screens/add-user/add-user-page.tsx

```diff
@@ -0,0 +1,60 @@
+import React, {useState} from 'react';
+import {Text, View} from 'react-native';
+import {ButtonComponent} from '../../components/button.component';
+import {InputComponent} from '../../components/input.component';
+import {birthDateIsValid, cpfHasValidLength} from '../login/login-validation';
+import {styleAddUser} from './add-user-page.styles';
+
+export const AddUserPage = () => {
+  const [name, setName] = useState('');
+  const [phone, setPhone] = useState('');
+  const [cpf, setCpf] = useState('');
+  const [birthDate, setBirthDate] = useState('');
+  const [email, setEmail] = useState('');
+  const [role, setRole] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const userValidation = () => {
+    if (
+      name === '' ||
+      email === '' ||
+      phone === '' ||
+      birthDate === '' ||
+      cpf === '' ||
+      role === ''
+    ) {
```

Comment:
em outro PR eu lembro que você fez uma função `isEmpty` (ou algo assim) né? será que daria pra usar aqui também? 🤔 
```suggestion
    if (isEmpty(name) || isEmpty(email) || ...) {
      // ...
    }
```
ou, se quiser deixar mais legal 😅
```suggestion
if ([
  name, 
  email, 
  phone, 
  birthDate, 
  cpf, 
  role,
].some(isEmpty)) { // considerando que exista aquela função 'isEmpty(string)'
  setErrorMessage("Campos não devem estar vazios");
}
```